posts/2020-02-27-Rust-1.41.1.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Announcing Rust 1.41.1"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team has published a new point release of Rust, 1.41.1.
+Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.41.1 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the appropriate page on our website.
+
+[install]: https://www.rust-lang.org/tools/install
+
+## What's in 1.41.1 stable
+
+Rust 1.41.1 addresses two critical regressions introduced in Rust 1.41.0:
+a soundness hole related to static lifetimes, and a miscompilation causing segfaults.
+These regressions are not affecting past releases of Rust,
```

Comment:
Well, that's not really true -- the latest reproducer on https://github.com/rust-lang/rust/issues/69225 goes back to 1.38, as long as we've been using LLVM 9.

Reply:
cc @pietroalbini

Reply:
Uh, woops.